core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/RetryPolicy.java

```diff
@@ -234,7 +234,7 @@ public int hashCode() {
         result = 31 * result + Boolean.hashCode(additionalRetryConditionsAllowed);
```

Comment:
> Not necessary since it's a primitive `boolean`:
> 
> https://github.com/aws/aws-sdk-java-v2/blob/3420a837ad9df731d176264f625030c223d61b06/core/sdk-core/src/main/java/software/amazon/awssdk/core/retry/RetryPolicy.java#L48

YEap I saw it later , too late to delete the comment